crypto/asn1/n_pkey.c

```diff
@@ -8,24 +8,21 @@
  */
 
 #include "openssl/opensslconf.h"
-#ifdef OPENSSL_NO_RSA
```

Comment:
This change does not seem to be reflected in the asn1/build.info. Also should the NO_RC4 be handled by the build.info as well?

Reply:
Fixed, and good point about the RC4 flag.